internal/pkg/cmd/prerunner.go

```diff
@@ -309,6 +313,10 @@ func (r *PreRun) Authenticated(command *AuthenticatedCLICommand) func(*cobra.Com
 			return err
 		}
 
+		if err := r.Config.DecryptCredentials(); err != nil {
```

Comment:
If this ends up staying in the prerunner (as opposed to being called before use like Brian suggested) then this call (and the one in AuthenticatedWithMDS below) might be redundant since both of these call `r.Anonymous`, which itself calls `r.Config.DecryptCredentials`.

Reply:
thanks for catching!